### ISSUE TITLE
File Handling API (revisited)

### ISSUE THREAD

**Issue by evanstade** [mozilla/standards-positions#629]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: File Handling API
* Specification or proposal URL:
  * https://github.com/WICG/file-handling/blob/main/explainer.md
  * https://github.com/WICG/manifest-incubations/pull/51
* Caniuse.com URL (optional): 
* Bugzilla URL (optional): 
* Mozillians who can provide input (optional): 

### Other information

Previous request (was deferred): https://github.com/mozilla/standards-positions/issues/158
Chromium implementation now complete and anticipating shipping soon.

**Comment by TodePond** [comment]
I would also like to request Mozilla's position on this. Apologies for the bump - but it seems like it's been a while since this issue was created.

The reason for this is that, as a developer, I would very much like to allow Firefox users to save and open files in a safe and user-friendly way.

If there is somewhere else that is better to discuss this, please do let me know.
Many thanks!

**Comment by mozfreddyb** [comment]
Is this still something that people pursue or did this wind up in https://wicg.github.io/file-system-access/?

Maybe we can close or rename this.

**Comment by tomayac** [comment]
> Is this still something that people pursue or did this wind up in https://wicg.github.io/file-system-access/?

This ended up in two places: [Manifest Incubations](https://wicg.github.io/manifest-incubations/#file_handlers-member) and [Web App Launch Handler API](https://wicg.github.io/web-app-launch/#dfn-launching-a-web-application-with-handling). It's a very useful API that Firefox might want to implement, now that installation of Web apps is possible again.

**Comment by saschanaz** [comment]
If it's split, should the s-p also split?

**Comment by tomayac** [comment]
> If it's split, should the s-p also split?

I don't think so. One isn't useful without the other, but [Web App Launch Handler API](https://wicg.github.io/web-app-launch/) is shared between File Handling (this very Issue) and [Launch Handler](https://developer.mozilla.org/en-US/docs/Web/API/Launch_Handler_API), which uses [`"launch_handler"`](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Manifest/Reference/launch_handler) from the Web App Manifest (this is https://github.com/mozilla/standards-positions/issues/567).
